src/main/java/uk/gov/hmcts/cmc/claimstore/services/staff/ClaimIssuedStaffNotificationService.java

```diff
@@ -28,91 +32,119 @@
     private final EmailService emailService;
     private final StaffEmailProperties staffEmailProperties;
     private final ClaimIssuedStaffNotificationEmailContentProvider provider;
-    private final PDFServiceClient pdfServiceClient;
-    private final SealedClaimContentProvider sealedClaimContentProvider;
-    private final DefendantPinLetterContentProvider defendantPinLetterContentProvider;
-    private final LegalSealedClaimContentProvider legalSealedClaimContentProvider;
+    private final LegalSealedClaimPdfService legalSealedClaimPdfService;
+    private final CitizenSealedClaimPdfService citizenSealedClaimPdfService;
+    private final DefendantPinLetterPdfService defendantPinLetterPdfService;
+    private final DocumentManagementService documentManagementService;
+    private final DocumentManagementSealedClaimHandler documentManagementSealedClaimHandler;
+    private final boolean documentManagementFeatureEnabled;
 
     @Autowired
     public ClaimIssuedStaffNotificationService(
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Constructor has 9 parameters, which is greater than 7 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.reform.hmcts.net/coding_rules#rule_key=squid%3AS00107)